plugins/tags/plugin.cpp

```diff
@@ -203,10 +205,18 @@ namespace golos { namespace plugins { namespace tags {
         boost::program_options::options_description&,
         boost::program_options::options_description& config_file_options
     ) {
+        config_file_options.add_options()
+            (
+                "tags-number", boost::program_options::value<uint16_t>()->default_value(5),
```

Comment:
uint8_t should be enough, it gives up to 255 tags. I know no case where such amount required